master/getting-started/openshift/reqs/system.md

```diff
@@ -0,0 +1,13 @@
+---
+title: System requirements
+canonical_url: 'https://docs.projectcalico.org/v3.0/getting-started/openshift/reqs/system'
+---
+
+{% include {{page.version}}/reqs-sys.md orch="OpenShift" %}
```

Comment:
I think we should just link to OpenShift's system requirements, and point out that we support anything they do (except Atomic / containerized) https://docs.openshift.com/container-platform/3.7/install_config/install/prerequisites.html

Reply:
It seems that they support more distros than we do, so that is why I call the ones that we support out here. For example, they support Fedora and also Atomic (as you note), whereas we do not. Let me know your thoughts on this tactic. 

The first link you provided does not even seem to list the supported distros, although we do see them in the second link.

Still, I think you are right, we should link to their system requirements for the additional information.